mobile/src/Student/screens/StudentGameIntro/index.js

```diff
@@ -122,7 +137,7 @@ const styles = StyleSheet.create({
     headerTeam: {
         color: colors.white,
         fontFamily: fontFamilies.montserratBold,
-        fontSize: fonts.xxLarge,
+        fontSize: 42,
```

Comment:
Lets add this to `theme.js` and use `fonts.` to fetch it. This helps us to have all font sizes in one place and add dynamic text sizing for accessibility later.